src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -236,14 +236,32 @@ protected Task Stop()
             RequestContext<object> requestContext)
         {
             if (helpParams == null) { helpParams = "get-help"; }
-
-            var psCommand = new PSCommand();
-            psCommand.AddCommand("Get-Help");
-            psCommand.AddArgument(helpParams);
-            psCommand.AddParameter("Online");
-
-            await editorSession.PowerShellContext.ExecuteCommand<object>(psCommand);
-
+            string script = @"
+function __Check-Help {
+    param (
+        $Command
+    )
+        $CommandExists = $false
+        Try {
+            Get-Command $Command -ErrorAction Stop
```

Comment:
Commands need to be module qualified (we've been burned on skipping that before) so this should be:

```powershell
Microsoft.PowerShell.Core\Get-Command $Command -ErrorAction Stop
```